apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/pom.xml

```diff
@@ -28,12 +28,21 @@
     <artifactId>kafka-reporter-plugin</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
+        <shade.com.google.source>com.google</shade.com.google.source>
```

Comment:
We don't need this. The property name is even longer than the original package name, there is no point.

Reply:
And, I don't think we are expecting shading all `com.google`. You are just going to shade GSON relative things. Please be clear about that.

Reply:
> And, I don't think we are expecting shading all `com.google`. You are just going to shade GSON relative things. Please be clear about that.

Let me optimize this.